celer/group_fast.pyx

```diff
@@ -418,8 +418,11 @@ cpdef celer_grp(
                                     &inc) / lc_groups[g]
                     norm_wg += w[j] ** 2
                 norm_wg = sqrt(norm_wg)
-                bst_scal = max(0.,
+                if norm_wg != 0.:
+                    bst_scal = max(0.,
```

Comment:
bst_scal is for BockSofthThresholding scaling : the formula for the BST of `wg` at level `lambda` is:
wg * max(0, 1 - lambda / norm(wg))     aka 0 if norm(wg) < lambda, and (1 - lambda/norm(wg)) wg otherwise 

I'm curious, how do you end up with a vanishing `wg` after a gradient step ? This should not happen with probability 1, I'm guessing a 0 group X_g ?